.idea/inspectionProfiles/Project_Default.xml

```diff
@@ -0,0 +1,6 @@
+<component name="InspectionProjectProfileManager">
```

Comment:
could you please remove `.idea` files?
Feel free to add `.idea` dir into the `.gitignore` if it helps you.

Reply:
Done! Thank you for noticing. I migrated to Visual Studio and it was my first commit ;)